addons/gdUnit4/test/spy/UnitPreset.gd

```diff
@@ -1,18 +1,18 @@
-class_name UnitPreset
+class_name ClassWithEnumConstructor
```

Comment:
When you rename the class you should also rename the file

Reply:
You are right, I renamed the file.